File: Leetcode/javascript/jewels-and-stones.js
// https://leetcode.com/problems/jewels-and-stones/



var sLen = stones.length
var jCount = 0
for (i= 0; i < sLen; i++) {
    if (jewels.includes(stones[i])  === true) {
        console.log("Jewel")
        jCount += 1
    }
    else{ console.log("Not a Jewel")}
}
return jCount

File: Leetcode/python/jewels-and-stones.py
# https://leetcode.com/problems/jewels-and-stones/

class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        print(jewels)
        print(stones)
        sLen = len(stones)
        jCount = 0
        for i in range(sLen):
            if stones[i] in jewels:
                print("Jewel")
                jCount += 1
            else:
                print("Not a Jewel")
        return jCount